releases/stellar/2025-03-GMP-v1.1.0.md

```diff
@@ -56,75 +56,93 @@ node stellar/faucet.js
+| **Mainnet**          | `'[("axelar","0xe75bfad0ac5c972ac4053d11bade19d182c4799f22872c3cca8090e07a229a5f-250959"),("axelar","0x7ed28ebf275f430b64487ac74f944b151edf3b8392456d789f04f83bf75c079e-257089"),("axelar","0xfecb1bbe5e1eafa7fefb519884e3d58a05066c7ca07054372cab5a8105b31518-250301"),("axelar","0x6538b066d98bbd9d4e5338f19146d773747022fc4e698376671e4a1d228e69e3-252142"),("stellar","0x6a7348f84b5c0a42892656228834fcf3880a827cedeafcf6cfc171d825250395-2"),("axelar","0x93f6b146c47fe45c325b804e559fbb9036eba114ebb7a53ab12796aa5d5ba50a-256802"),("axelar","0x49ad7fd5f17a11694f1d0abdc1c498eed6f7128159685e3c066b6d1e4a3928fe-253098"),("axelar","0x85f5f5df8c774da7a50902f295fc9c9643187ab1bab4ae0d76dcfc11bd36bbc4-257244"),("axelar","0xd43f92c82e733db3d381addb7d8cff2f5d721e4e4f976f7811df1835104373b0-256938"),("axelar","0x621b48ce71ad7841772436ce1e5ed96238e4e537bbf37e55fdcc19e5ee3f6b4f-256521"),("stellar","0x965bd66495ad46390b97b6c03e4e52abe77b213cbaedfbabbd9e96b74648f847-2"),("axelar","0xb0f33127bb7049f967df74df92094ce8f9c32a21b33b048ba4bc34306ba08063-251212"),("axelar","0xe9a205b406e320b3124cb2e171567105fab78ac980d7f5dcc41a407dd955a308-251084")]'` |
 
-5. Upgrade the `AxelarGateway` contract & migrate storage schema
+5. Pause the `AxelarGateway` contract
 
 ```bash
-node stellar/deploy-contract.js upgrade AxelarGateway --version 1.1.0 --migration-data '[("<sourceChainN>", "<messageIdN>")]'
+node stellar/contract.js pause AxelarGateway
 ```
 
-6. Retrieve full list of operators used by the relayer
+6. Verify `AxelarGateway` is paused
 
-| Network              | `AxelarOperators storage`                                  |
-| -------------------- | ---------------------------------------------------------- |
-| **Devnet-amplifier** | [source](https://stellar.expert/explorer/testnet/contract/CABRZHKAJVJFX5IZWL7KPA6ZEX6I347W7GBXAUX6WVOGUUBRWCDGOUSI/storage)                   |
-| **Stagenet**         | `TBD`                                                      | // TODO: ADD
-| **Testnet**          | `TBD`                                                      | // TODO: ADD
-| **Mainnet**          | `TBD`                                                      | // TODO: ADD
+```bash
+node stellar/contract.js paused AxelarGateway
+```
+
+7. Upgrade the `AxelarGateway` contract & migrate storage schema
+
+```bash
+node stellar/deploy-contract.js upgrade AxelarGateway --version 1.1.0 --migration-data '<[("<sourceChainN>", "<messageIdN>")]>'
+```
+
+8. Unpause the `AxelarGateway` contract
+
+```bash
+node stellar/contract.js unpause AxelarGateway
+```
+
+9. Retrieve full list of operators used by the relayer
+
+| Network              | `AxelarOperators storage`                                           |
+| -------------------- | ------------------------------------------------------------------- |
+| **Devnet-amplifier** | [source](https://stellar.expert/explorer/testnet/contract/CDOWLXRHI3JVLTDXUZK4AA5RYT4MO53KGGHWRVTMLI6C74HJPAG5W7WO/storage) |
```

Comment:
in the future, we could look into querying the explorer API dynamically to get indexed storage, if we need to do a similar migration